src/sentry/api/endpoints/organization_dashboards.py

```diff
@@ -163,12 +165,31 @@ def get(self, request: Request, organization) -> Response:
             order_by = ["last_visited" if desc else "-last_visited"]
 
         elif sort_by == "mydashboards":
```

Comment:
Can we feature flag this so it doesn't rollout until the table also rolls out? So current users aren't affected and it'll make more sense with the table.

Also, should this get a different sort key? Instead of `mydashboards` it should be something like `creator` now, since it doesn't prioritize the request user anymore

Reply:
I think it makes more sense to be under myDashboards since it's still prioritizing the request user's dashboards and only changing the order of the remaining dashboards

Reply:
OH oops, yeah, you're right. I misread the annotation as the sort, sounds good.